handsontable/test/e2e/core/alter/insertColEnd.spec.js

```diff
@@ -653,6 +653,114 @@ describe('Core.alter', () => {
         expect(getDataAtRow(0)).toEqual(['E1', null, 'D1', null, 'C1', 'B1', 'A1', null]);
         expect(getSourceDataAtRow(0)).toEqual(['A1', null, 'B1', 'C1', 'D1', null, 'E1', null]);
       });
+
+      it('should insert column at proper position when column index sequence is shifted', async() => {
+        handsontable({
+          data: createSpreadsheetData(5, 5)
+        });
+
+        columnIndexMapper().setIndexesSequence([4, 0, 1, 2, 3]);
+
+        await alter('insert_col_end', 1, 1);
+
+        expect(getDataAtRow(0)).toEqual(['E1', 'A1', null, 'B1', 'C1', 'D1']);
+        expect(getSourceDataAtRow(0)).toEqual(['A1', null, 'B1', 'C1', 'D1', 'E1']);
+
+        await alter('insert_col_end', 0, 1);
+
+        expect(getDataAtRow(0)).toEqual(['E1', null, 'A1', null, 'B1', 'C1', 'D1']);
+        expect(getSourceDataAtRow(0)).toEqual(['A1', null, 'B1', 'C1', 'D1', 'E1', null]);
+
+        await alter('insert_col_end', 6, 1);
+
+        expect(getDataAtRow(0)).toEqual(['E1', null, 'A1', null, 'B1', 'C1', 'D1', null]);
+        expect(getSourceDataAtRow(0)).toEqual(['A1', null, 'B1', 'C1', 'D1', null, 'E1', null]);
+      });
+
+      it('should not copy column filters from the source column to the new one', async() => {
```

Comment:
Core doesn't know anything about plugins. Moving _Filters_ related tests to the filter's tests directory would be much better.

Reply:
You're right. I moved it.